scenarios/tracking/02_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<ul><li>Bit confusing right now. Needs a bit more test/explanations, e.g. are those results for the training or test set.</li><li>Sentence "which matches the reported accuracy" could be more precise. E.g. can we add a screenshot from the uploaded results to the evaluation server, to also show what the output of the evaluation server looks like? Might also need some code example how to get the results file which needs to be uploaded. Could have two sections: ##evaluate on training set and ##evaluate on test set.</li><li>[minor] can you please remove the 00, 01, 02 before each section. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>